doc/en/user/source/styling/mbstyle/cookbook/points.rst

```diff
@@ -0,0 +1,126 @@
+.. _mbstyle_cookbook.points:
+
+Points
+======
+
+While points are seemingly the simplest type of shape, possessing only position and no other dimensions, MBStyle has a ``circle`` type that can be styled to represent a point.

```

Comment:
After the other changes to the sentance, the "While" no longer makes much sense here